code/modules/mob/living/carbon/human/npc/dungeon_pack_npc.dm

```diff
@@ -0,0 +1,768 @@
+/mob/living/carbon/human/species/human/northern/dungeon_base
+	aggressive = TRUE
+	mode = AI_IDLE
+	gender = MALE
+	faction = list("dungeon")
+	ambushable = FALSE
+	dodgetime = 20
+	defprob = 35
+	flee_in_pain = FALSE
+	wander = TRUE
+
+	var/combat_loop_running = FALSE
+	var/next_cast = 0
+
+/mob/living/carbon/human/species/human/northern/dungeon_base/examine(mob/user)
+	to_chat(user, "You don't recognize this person.")
+
+/mob/living/carbon/human/species/human/northern/dungeon_base/Initialize()
+	. = ..()
+	set_species(/datum/species/human/northern)
```

Comment:
aren't these a subtype of `/mob/living/carbon/human/species/human/northern`? is this even necessary?